server/server.js

```diff
@@ -0,0 +1,30 @@
+// Server
+const express = require('express')
+const app = express()
+const port = 5000
+const path = require('path');
+
+
+app.listen(port, () => {
```

Comment:
Could seperate the logic for starting server into another file to make it cleaner to run and stop the server launching if you were to add tests for this file